src/components/UserProperties.vue

```diff
@@ -46,35 +43,25 @@ export default {
 
     updateUserConfigs(userConfigs) {
       const jsonUserConfig = JSON.stringify({
```

Comment:
Why not move this logic also to to the `UserService` and just let `UserService.updateUserConfig` handle all the stuff? Same for other methods

Reply:
Same comment as below